src/main/kotlin/com/openlattice/apps/services/AppService.kt

```diff
@@ -118,7 +116,7 @@ class AppService(
     }
 
     fun deleteRoleFromApp(appId: UUID, roleId: UUID) {
-        val app = getApp(appId)
+        getApp(appId)
```

Comment:
should probably rename to something more meaningful (re: using as a null check)
Or just use
```
        requireNotNull(appId) { "App with id $appId does not exist." }
```
though actually its not nullable so. . . can just remove this line

Reply:
This is just checking if app exists right? Seems like there's probably an unnecessary deep load of app going on here and we should have a method that just does containsKey on the app map.